lib/reporting/identity_verification_report.rb

```diff
@@ -286,6 +339,8 @@ def query
           , properties.event_properties.success = '0' AND properties.event_properties.doc_auth_result NOT IN ['Failed', 'Attention'] AS doc_auth_failed_non_fraud
         | fields
             !fraud_review_pending and !gpo_verification_pending and !in_person_verification_pending and !has_other_deactivation_reason AS identity_verified
+        | fields
+            properties.event_properties.ab_tests AS ab_tests
```

Comment:
So I just ran an example query in the terminal for this, and I think this will not work, Cloudwatch doesn't return the whole hash for these. Based on this, we'd expect both `event_properties` hash and `success` but `event_properties` is missing:

```ruby
# aws-vault exec prod-power -- bundle exec rails c
require 'reporting/cloudwatch_client'
Reporting::CloudwatchClient.new.fetch(
  query: <<~STR,
fields properties.event_properties, properties.event_properties.success
| filter ispresent(properties.event_properties.success)
| limit 10
STR
  from: 30.minutes.ago,
  to: 10.minutes.ago,
)
=> [{"properties.event_properties.success"=>"1"},
 {"properties.event_properties.success"=>"1"},
 {"properties.event_properties.success"=>"1"},
 {"properties.event_properties.success"=>"1"},
 {"properties.event_properties.success"=>"1"},
 {"properties.event_properties.success"=>"1"},
 {"properties.event_properties.success"=>"1"},
 {"properties.event_properties.success"=>"1"},
 {"properties.event_properties.success"=>"1"},
 {"properties.event_properties.success"=>"1"}]
```

Reply:
Confirmed in the CW UI, entire hash responses don't show up.

<img width="613" alt="Screenshot 2024-02-08 at 2 49 25 PM" src="https://github.com/18F/identity-idp/assets/458784/7db7318e-2c46-48c9-bc61-514ced977528">

We could get this by requesting and parsing `@message` but that's going to be a lot of very large payloads, which has lots of memory implications for running this, so if we did that, I would push strongly for finding a way to make AB test parsing optional

Reply:
Ugh, thanks for catching this! 

I'm thinking now of leaving them under `ab_tests`, but then taking your suggestion for separate classes to process data along with an explict list of ab_tests to process rather than the desired automagic. Then if we want this data for a test, we enable/disable it.